plugin/advisor/pg/advisor_naming_unique_key_convention.go

```diff
@@ -35,6 +35,9 @@ func (*NamingUKConventionAdvisor) Check(ctx advisor.Context, statement string) (
 		return nil, err
 	}
 
+	// Naming length limit feature is limited for PG, the PG parser will auto slice the name to make sure its length <= 63
+	// Reference:
+	// https://www.postgresql.org/docs/current/limits.html
```

Comment:
I guess we don't need comment here, the constant in sql_review.go is good enough.